development/readme_eclipse_4.32.html

```diff
@@ -1104,7 +1104,7 @@ <h2 id="RunningEclipse">
   <p>
     <a
       class="mozTocH2"
-      name="mozTocId19817"> After installing the Eclipse SDK in a directory, you can start the Workbench by running the Eclipse executable included with the release (you also need a Java SE 17 JRE,
```

Comment:
Just like previous one.

Reply:
I extended the text a bit and linked to the "target environments" section where the difference is explained, does it sound good now?